### ISSUE TITLE
[feature request]  1.10.2 config option to turn off Flugel Tiara

### ISSUE THREAD

**Issue by Zygus42** [williewillus/Botania#463]
1.10.2 please add config option to turn off Flugel Tiara.  Other config options around this item that would be nice to have would be to be able to (a) adjust the base flight speed and (b) the amount of mana it costs to use the item.

**Comment by quat1024** [comment]
For disabling it, you can use something like MineTweaker to remove its recipe.

**Comment by Zygus42** [comment]
Yep - and I did just add CraftTweaker and ModTweaker (together they're the current 1.10 version of MineTweaker) in just to remove the EFLN in Tinker's Construct, so I might as well.  But if every mod had options to customize everything, we could remove 2 mods from our already crowded build - will never happen I know, but I can dream !

What I would really like for our world is a mod to set the max flight and run speeds to given (low) values in order to make roads, riverways and railways useful enough in the mid to late game to actually build instead of people pushing for flight and teleportation.  I've chucked it out there as a mod suggestion already, but not much response.
